### DISCUSSION TITLE
Issues with alpine click and livewire click.

### DISCUSSION THREAD

**Discussion by chistel** [livewire/livewire#4647] [1. Help]
Good day, am having an issue with livewire and alpinejs, where click event from livewire triggers the alpinejs click event.

I currently use alpinejs 3 and the current livewire version.

I attempted using `x-on:click="$wire.openSecondaryMenu('{{ $sideMenu['name'] }}')"`, but i get undefined error

here's the livewire component.
`<div>

    <aside :class="(mobile() && isSidebarOpen ? 'fixed inset-y-0 block' : 'hidden md:flex')"
           class="mt-16 md:mt-0 flex-shrink-0 z-20"
    >
        <nav class="primary-menu w-60 bg-primary border-r min-h-screen overflow-y-auto">
           
            <div class="md:py-2">

                <ul class="flex flex-col w-full" id="menu-{{ $primaryMenus['name'] }}">

                    @if(isset($primaryMenus['children']) && count($primaryMenus['children']))
                        @foreach($primaryMenus['children'] as $i => $sideMenu)
                            <li id="primary-menu-{{ $sideMenu['name'] }}"
                                wire:key="primary-menu-{{ $sideMenu['name'] }}">
                                <a
                                    @if (isset($sideMenu['link']))
                                    href="{{ $sideMenu['link'] }}"
                                    @elseif ($sideMenu['children'])
                                    wire:click.prevent="openSecondaryMenu('{{ $sideMenu['name'] }}')"
                                    @endif
                                    class="cursor-pointer flex flex-row items-center mb-1 py-3 px-3 text-white hover:bg-white hover:text-primary {{ $sideMenu['active'] ? 'bg-white text-primary' : '' }}">
                                    @if ($sideMenu['icon'])
                                        <i class="fas fa-{{ $sideMenu['icon'] }}"></i>
                                    @endif
                                    <span class="ml-3">{{ $sideMenu['text'] }}</span>
                                </a>
                            </li>
                    @endforeach
                @endif
                </ul>
            </div>
        </nav>
    </aside>

</div>

`

**Comment by joshhanley** [comment]
@chistel can you share a screenshot of the error? I can't see your `x-on:click` in the component code, is it missing something?
